homeassistant/components/recorder/db_schema.py

```diff
@@ -588,6 +589,10 @@ def shared_attrs_bytes_from_event(
                 *ALL_DOMAIN_EXCLUDE_ATTRS,
                 *state_info["unrecorded_attributes"],
             }
+            if MATCH_ALL in state_info["unrecorded_attributes"]:
+                for attribute in state.attributes:
```

Comment:
Should we exclude common attributes from removal eg. `friendly_name` and others?

Reply:
If I recall correctly logbook, recorder, etc only care about friendly name, state class, and device class